libcockatrice_deck_list/libcockatrice/deck_list/deck_list.h

```diff
@@ -223,32 +235,38 @@ public slots:
     ~DeckList() override;
 
     /// @name Metadata getters
+    /// The individual metadata getters still exist for backwards compatibility.
+    /// TODO: Figure out when we can remove them.
```

Comment:
we can remove them as soon as no functions use them anymore, I don't see any reason not to